en/contributing/code.rst

```diff
@@ -12,7 +12,7 @@ Before working on patches for CakePHP, it's a good idea to get your environment
 setup. You'll need the following software:
 
 * Git
-* PHP 5.4.16 or greater
+* PHP 5.5.8 or greater
```

Comment:
Just curious, could we repalce the literal for the required PHP version with a constant or variable?